app.js

```diff
@@ -8,18 +8,35 @@ const app = express();
 const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
+
 app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
-
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+// Route to Beers
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+  .then((beersFromApi) => {res.render('beers', { beersFromApi });
+    console.log('Beers from the database: ', beersFromApi)
```

Comment:
Please remove 'console.log()' after you're done confirming that something works.